trackers/core/deepsort/kalman_box_tracker.py

```diff
@@ -11,10 +11,45 @@ class DeepSORTKalmanBoxTracker(SORTKalmanBoxTracker):
     tracked object (bounding box), with a Kalman filter to predict and update
     its position. It also maintains a feature vector for the object, which is
     used to identify the object across frames.
+
```

Comment:
`DeepSORTKalmanBoxTracker` still inherit from `SORTKalmanBoxTracker`, and `SORTKalmanBoxTracker` is still imported into the file